src/io.jl

```diff
@@ -0,0 +1,130 @@
+# Input/ Output of Unfold results
+"""
+    save_results(results::DataFrame, bids_root::String; 
+        save_folder::String="Unfold",
+        overwrite::Bool=false)
+
+Function to save unfold models in your BIDS root folder. Automatically creates a save_folder (default = "Unfold") in the derivatives and subsequentely safes each model in results according to BIDS.
+
+If overwrite is false the function will check for already saved results in the same folder and with the same name i.e. same subject/session/task/run.
+"""
+function save_results(results::DataFrame, bids_root::String;
+    save_folder::String="Unfold",
+    overwrite::Bool=false)
+
+    # Make folder to save in
+    save_in = joinpath(bids_root, "derivatives", save_folder)
+    if !isdir(save_in) && isdir(bids_root)
+        mkdir(save_in)
+    elseif !isdir(bids_root)
+        throw("$bids_root does not exists.")
+    end
+
+    for row in eachrow(results)
+
+        # Make folder for subject
+        tmp_folder = joinpath(save_in, "sub-" * row.subject, "eeg")
+        mkpath(tmp_folder)
+
+        # Make a filename based on available data
+        file_name = "sub-" * row.subject
+        if !ismissing(row.ses)
+            file_name = file_name * "_ses-" * row.ses
+        end
+        if !ismissing(row.task)
+            file_name = file_name * "_task-" * row.task
+        end
+        if !ismissing(row.run)
+            file_name = file_name * "_run-" * row.run
+        end
+        file_name = file_name * "_eeg.jld2"
+
+        fullfile_path = joinpath(tmp_folder, file_name)
+        if !overwrite && !isfile(fullfile_path)
+            save(fullfile_path, row.model; compress=true)
+        elseif !overwrite && isfile(fullfile_path)
+            @warn("overwrite is set to false and at least one subject has already saved results in the folder $save_in
+            If you're sure you want to overwrite your data, please set overwrite=true   
+            Subject file: $file_name")
+            return
+        else
+            save(fullfile_path, row.model; compress=true)
+        end
+    end
+
+end
+
+"""
+    function load_results()
+
+Load Unfold models existing in a BIDS root folder. If lazy is true (default) then data will not be loaded and a DataFrame containing all file names + information will be returned (similar to bids_layout output).
```

Comment:
the keyword-arguments are not explained.

especially `lazy` is unclear,

- why is the argument name "save_folder" if you load something? I would call it either something neutral in all functions, or load_folder. But now looking at it, it is the name of the derivatives, so maybe `derivatives_subfolder` - or whatever BIDS decided to call these folders ;)

Reply:
Added the arguments now, with explanations. However, I didn't change the explanation for the lazy keyword, since I thought that was clear... Can you describe more what is unclear?

Reply:
ah sorry. I think I expected something like:
```
# keywords
`lazy` (boolean:true): Do not actually load the dataset, only return a dataframe with paths
```

Reply:
UnfoldMakie has good docstrings in general, maybe something to copy as a template

Reply:
Will do, but in a later PR. For now let's add the I/O functionality